File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rsp/contentRecovery/api/ContentRecoveryImpl.java
package org.ngbp.jsonrpc4jtestharness.rsp.contentRecovery.api;

import org.ngbp.jsonrpc4jtestharness.rsp.contentRecovery.model.CecoveredComponentInfo;
import org.ngbp.jsonrpc4jtestharness.rsp.contentRecovery.model.ContentRecoveryState;

public class ContentRecoveryImpl implements ContentRecovery {

    @Override
    public ContentRecoveryState queryContentRecoveryState() {
        return null;
    }

    @Override
    public CecoveredComponentInfo queryRecoveredComponentInfo() {
        return null;
    }

    @Override
    public CecoveredComponentInfo contentRecoveryStateChangeNotification() {
        return null;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/contentRecovery/ContentRecoveryImpl.kt
package org.ngbp.jsonrpc4jtestharness.rpc.contentRecovery

import org.ngbp.jsonrpc4jtestharness.rpc.contentRecovery.model.ContentRecoveryState
import org.ngbp.jsonrpc4jtestharness.rpc.contentRecovery.model.RecoveredComponentInfo

class ContentRecoveryImpl : IContentRecovery {
    override fun queryContentRecoveryState(): ContentRecoveryState {
        return ContentRecoveryState()
    }

    override fun queryRecoveredComponentInfo(): RecoveredComponentInfo {
        return RecoveredComponentInfo()
    }
}